code/go/0chain.net/smartcontract/common/wallet.go

```diff
@@ -0,0 +1,41 @@
+package common
+
+import (
+	"encoding/hex"
+
+	"0chain.net/chaincore/config"
+	"0chain.net/core/common"
+	"0chain.net/core/encryption"
+	"github.com/0chain/common/core/logging"
+)
+
+
+func ValidateWallet(publicKey, delegateWalletID string) *common.Error {
```

Comment:
Rename to `ValidateDelegateWallet()` and add comments to describe what it does, we need to ensure it would not be misused by users, i.e thought the the function will return nil if the clientID and pubkey matches